app/src/main/kotlin/org/cru/godtools/ui/dashboard/lessons/LessonFilters.kt

```diff
@@ -0,0 +1,140 @@
+                    .weight(1f)
+                    .align(Alignment.CenterVertically)
+            )
+        }
+    }
+}
+
+@OptIn(ExperimentalMaterial3Api::class, ExperimentalFoundationApi::class)
+@Composable
+internal fun LessonLanguageFilter(modifier: Modifier = Modifier, viewModel: LessonsViewModel = viewModel()) {
+    val context = LocalContext.current
+    val selectedLanguage by viewModel.selectedLanguage.collectAsState()
+
+    val lessonLanguages by viewModel.filteredLanguages.collectAsState(emptyList())
+    var expanded by remember { mutableStateOf(false) }
+    var query by remember { mutableStateOf("") }
+    val scope = rememberCoroutineScope()
+
+    ElevatedButton(
+        onClick = { expanded = true },
+        modifier = modifier.semantics { role = Role.DropdownList }
+    ) {
+        Text(
+            selectedLanguage.getDisplayName(context, LocalAppLanguage.current),
+            maxLines = 1,
+            overflow = TextOverflow.Ellipsis,
+            modifier = Modifier.weight(1f),
+        )
+        ExposedDropdownMenuDefaults.TrailingIcon(expanded = expanded)
+        LazyDropdownMenu(
+            expanded = expanded,
+            onDismissRequest = { expanded = false },
+            modifier = Modifier.sizeIn(maxHeight = DROPDOWN_LESSON_MAX_HEIGHT, maxWidth = DROPDOWN_LESSON_MAX_WIDTH)
+        ) {
+            stickyHeader {
+                Surface(color = MaterialTheme.colorScheme.surface) {
+                    SearchBar(
+                        query = query,
+                        onQueryChange = { change ->
+                            scope.launch {
+                                query = change
+                                viewModel.updateQuery(change)
+                            }
+                        },
+                        onSearch = {},
+                        active = false,
+                        onActiveChange = {},
+                        colors = GodToolsTheme.searchBarColors,
+                        placeholder = {
+                            Text(stringResource(R.string.language_settings_downloadable_languages_search))
+                        },
+                        content = {},
+                        modifier = Modifier
+                            .padding(horizontal = 16.dp, vertical = 16.dp)
+                            .fillMaxWidth()
+                            .wrapContentWidth()
+                    )
+                }
+            }
+
+            items(lessonLanguages, key = { (lang) -> lang.code }) { (lang, count) ->
+                HorizontalDivider(Modifier.padding(start = ListItemStartPadding, end = ListItemStartPadding))
+                FilterMenuItem(
+                    label = { LanguageName(lang) },
+                    supportingText = pluralStringResource(
+                        R.plurals.dashboard_tools_section_filter_available_tools,
+                        count,
+                        count,
+                    ),
```

Comment:
The phrase for this dropdown should be "Lessons available" and not "Tools available"
![Screenshot 2024-08-13 at 9 36 30 AM](https://github.com/user-attachments/assets/b70e483d-284e-41b9-8a8c-40e59a994c75)